src/Fixer/LanguageConstruct/NoUnsetOnPropertyFixer.php

```diff
@@ -33,9 +33,9 @@ public function getDefinition()
             'Properties should be set to `null` instead of using `unset`.',
             [new CodeSample("<?php\nunset(\$this->a);\n")],
             null,
-            'Changing variables to `null` instead of unsetting them will mean they still show up '.
-            'when looping over class variables. With PHP 7.4, this rule might introduce `null` assignments to '.
-            'property whose type declaration does not allow it.'
+            'Risky when relying on attributes to be removed using `unset` rather than be set to `null`.'.
+            ' Changing variables to `null` instead of unsetting means these still show up when looping over class variables.'.
```

Comment:
```suggestion
            ' Changing variables to `null` instead of unsetting means these still show up when looping over class variables and reference properties remains unbroken.'.
```

Reply:
thanks added